src/main/java/org/influxdb/impl/RequestAuthenticator.java

```diff
@@ -0,0 +1,39 @@
+package org.influxdb.impl;
+
+import okhttp3.Authenticator;
+import okhttp3.Credentials;
+import okhttp3.Request;
+import okhttp3.Response;
+import okhttp3.Route;
+
+import java.io.IOException;
+
+public class RequestAuthenticator implements Authenticator {
```

Comment:
No a single test covering this new code, without this we cant accept this PR.

Reply:
Would a JUnit test be sufficient? Like this one:
https://github.com/essagl/influxdb-java/blob/master/src/test/java/org/influxdb/impl/RequestAuthenticatorTest.java